npc_simulator/src/node.cpp

```diff
@@ -153,25 +164,25 @@ void NPCSimulator::mainTimerCallback()
       // update velocity x
       updateVelocity(&obj, delta_time);
 
-      //calculate future position
-      npc_simulator::msg::Object * future_obj = new npc_simulator::msg::Object(obj);  //deep copy
+      // calculate future position
+      npc_simulator::msg::Object * future_obj = new npc_simulator::msg::Object(obj);  // deep copy
```

Comment:
@yamacir-kit clang-tidy pointed out this could be a memory leak. Can this be made a stack object?

Reply:
Made into a stack object